test/integration/integration.test.ts

```diff
@@ -202,15 +214,86 @@ describe('Integration tests', (): void => {
       const outgoingPayment = await createOutgoingPayment(
         senderWalletAddress,
         finalizedGrant,
-        quote
+        {
+          metadata: {},
+          quoteId: quote.id
+        }
       )
-      await getOutgoingPayment(
+      const outgoingPayment_ = await getOutgoingPayment(
         outgoingPayment.id,
-        finalizedGrant,
-        amountValueToSend
+        finalizedGrant
       )
+
+      expect(outgoingPayment_.receiveAmount.value).toBe(amountValueToSend)
+      expect(outgoingPayment_.sentAmount.value).toBe(amountValueToSend)
+
       await getPublicIncomingPayment(incomingPayment.id, amountValueToSend)
     })
+    test('Open Payments without Quote', async (): Promise<void> => {
+      const {
+        grantRequestIncomingPayment,
+        createIncomingPayment,
+        grantRequestOutgoingPayment,
+        pollGrantContinue,
+        createOutgoingPayment,
+        getOutgoingPayment
+      } = testActions.openPayments
+      const { consentInteraction } = testActions
+
+      const receiverWalletAddressUrl =
+        'https://happy-life-bank-test-backend:4100/accounts/pfry'
+      const senderWalletAddressUrl =
+        'https://cloud-nine-wallet-test-backend:3100/accounts/gfranklin'
+
+      const receiverWalletAddress = await c9.opClient.walletAddress.get({
+        url: receiverWalletAddressUrl
+      })
+      expect(receiverWalletAddress.id).toBe(receiverWalletAddressUrl)
+
+      const senderWalletAddress = await c9.opClient.walletAddress.get({
+        url: senderWalletAddressUrl
+      })
+      expect(senderWalletAddress.id).toBe(senderWalletAddressUrl)
+
+      const debitAmount = {
+        assetCode: senderWalletAddress.assetCode,
+        assetScale: senderWalletAddress.assetScale,
+        value: '500'
+      }
+
+      const incomingPaymentGrant = await grantRequestIncomingPayment(
+        receiverWalletAddress
+      )
+      const incomingPayment = await createIncomingPayment(
+        receiverWalletAddress,
+        incomingPaymentGrant.access_token.value
+      )
+
+      const outgoingPaymentGrant = await grantRequestOutgoingPayment(
+        senderWalletAddress,
+        {
+          debitAmount,
+          receiveAmount: debitAmount
+        }
+      )
+      await consentInteraction(outgoingPaymentGrant, senderWalletAddress)
+      const grantContinue = await pollGrantContinue(outgoingPaymentGrant)
+      const outgoingPayment = await createOutgoingPayment(
+        senderWalletAddress,
+        grantContinue,
+        {
+          incomingPayment: incomingPayment.id,
+          debitAmount
+        }
+      )
+
+      const outgoingPayment_ = await getOutgoingPayment(
+        outgoingPayment.id,
+        grantContinue
+      )
+
+      expect(outgoingPayment_.debitAmount).toMatchObject(debitAmount)
```

Comment:
should we also check for sentAmount here like we do in the above test?

Reply:
I didn't add it originally because we are not defining it ourselves, so it may be hard to anticipate. But we ran into that with another test and handled it. Maybe we can do the same thing here - I will look into that.